src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const action = actions[i];
+
+    if (action.type === 'addProperties') {
```

Comment:
please use `switch/case` construction. do not forget about `default` case
![image](https://github.com/mate-academy/js_stateful-object/assets/22824431/2b005b9a-66c5-44ed-a7c3-89e3210a65f6)